nncf/quantization/quantize_model.py

```diff
@@ -501,9 +501,10 @@ def compress_weights(
                 f"but given {mode.value} mode."
             )
 
-        if any((awq, scale_estimation, gptq, lora_correction)):
+        if any((awq, scale_estimation, gptq, lora_correction, dataset)):
```

Comment:
I would suggest to add a separate error message for the dataset option.

Reply:
Alright.

Reply:
Done!